site-builder/src/site/resource.rs

```diff
@@ -348,22 +396,42 @@ impl ResourceManager {
                 .expect("the path should not terminate in `..`"),
```

Comment:
Question (line 391): should we remove the content encoding as a parameter, and rely on the headers?

Reply:
Yes! Content encoding remained there only temporarily in order to not break stuff. 
Now it has been removed from the parameters. ✔️